metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -744,6 +753,19 @@ private TableInfo getFromTableServiceProxy(final QualifiedName name,
                         : connectorTableServiceProxy.get(name, getTableServiceParameters, useCache));
     }
 
+    private void deleteCommonViewStorageTable(final QualifiedName viewName,
+                                              final QualifiedName storageTableName) {
+
+        try {
+            log.warn("Deleting storage table: {} belonging to common view: {}",
+                    storageTableName, viewName);
+            deleteAndReturn(storageTableName, false);
```

Comment:
This is a recursive call from the view deletion, and will return since the flag=false